test/conformance/tests/test_sriov_operator.go

```diff
@@ -2389,6 +2465,67 @@ func setSriovOperatorSpecFlag(flagName string, flagValue bool) {
 	}
 }
 
+func setOperatorConfigLogLevel(level int) {
+	cfg := sriovv1.SriovOperatorConfig{}
+	err := clients.Get(context.TODO(), runtimeclient.ObjectKey{
```

Comment:
i think both get and update calls should be in the same "Eventually" call
if an update occurred (resource version change) then update call will fail no matter how many times u attempt to set it.
for it to work an update call needs to be passed with the latest resource version IIRC

Reply:
good point! Moving get and update to the same Eventually block